### DISCUSSION TITLE
Issue with Aggrid

### DISCUSSION THREAD

**Discussion by samuellleow** [elimintz/justpy#363] [General]
![image](https://user-images.githubusercontent.com/60358173/157188384-be778136-edb7-427b-888f-cf46a7445d72.png)

I am implementing a feature where I am continuously polling data from a database and if the polled data defers from the latest data, it will produce a QCard with ag grid within the card, while pushing back the existing cards as soon in the image above.

![image](https://user-images.githubusercontent.com/60358173/157189000-06885546-28ee-4e11-9441-11868de59127.png)

In short, how I am implementing this feature is whenever there is a change in data, i will delete the components in the page and push the new and existing QCards into the page again. 

Printing the QCards would give me the correct aggrid tables/data that should be printed but the frontend is not printed correctly.

![image](https://user-images.githubusercontent.com/60358173/157189248-4ed3a248-6ca7-4d8a-867c-629ce6021053.png)

I am also receiving an error coming from aggrid.js file which I am unsure why the error was produced.

Thank you in advanced for any help or tips!

**Comment by WolfgangFahl** [comment]
It would be helpful if you'd actually post your code instead of screenshots of the code.
